packages/components/src/components/carousel/carousel-utils.spec.ts

```diff
@@ -114,17 +114,18 @@ describe('getSlidesAndAddNamedSlots()', () => {
     child3.slot = 'description';
 
     host.append(child1, child2, child3);
-    expect(getSlidesAndAddNamedSlots(host)).toEqual([child1]);
+    expect(getSlidesAndAddAttributes(host)).toEqual([child1]);
   });
 
-  it('should add incremental slot="slide-x" attribute on each child', () => {
+  it('should add incremental slot="slide-x" and tabindex="0" attributes on each child', () => {
     const host = document.createElement('p-carousel');
     const children = getChildren();
     host.append(...children);
 
-    const result = getSlidesAndAddNamedSlots(host);
+    const result = getSlidesAndAddAttributes(host);
     result.forEach((child, i) => {
       expect(child.slot).toBe(`slide-${i}`);
+      expect(child.tabIndex).toBe(0);
```

Comment:
Test name doesn't match what is happening here since here we check properties, not attributes.